src/MultiShapeletBasis.cc

```diff
@@ -109,8 +109,8 @@ MultiShapeletFunction MultiShapeletBasis::makeFunction(
     for (Iterator i = begin(); i != end(); ++i) {
         result.getComponents().push_back(ShapeletFunction(i->getOrder(), HERMITE, ellipse));
         result.getComponents().back().getEllipse().getCore().scale(i->getRadius());
-        result.getComponents().back().getCoefficients().asEigen()
-            = i->getMatrix().asEigen() * coefficients.asEigen();
+        ndarray::asEigenMatrix(result.getComponents().back().getCoefficients())
+            = ndarray::asEigenMatrix(i->getMatrix()) * ndarray::asEigenMatrix(coefficients);
```

Comment:
I realize this is outside the scope of the ticket, but do you even need Eigen here?

Reply:
I think is being used for the matrix multiplication